cwms-data-api/src/main/java/cwms/cda/data/dto/Project.java

```diff
@@ -0,0 +1,317 @@
+package cwms.cda.data.dto;
+
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import cwms.cda.api.errors.FieldException;
+import cwms.cda.formatters.Formats;
+import cwms.cda.formatters.annotations.FormattableWith;
+import cwms.cda.formatters.json.JsonV2;
+import java.time.Instant;
+
+@JsonDeserialize(builder = Project.Builder.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+@FormattableWith(contentType = Formats.JSON, formatter = JsonV2.class)
+public class Project extends CwmsDTO {
+
+    private final String name;
+    private final Double federalCost;
+    private final Double nonFederalCost;
+    private final Instant costYear;
+    private final String costUnit;
+    private final Double federalOAndMCost;
+    private final Double nonFederalOAndMCost;
+    private final String authorizingLaw;
+    private final String projectOwner;
+    private final String hydropowerDesc;
+    private final String sedimentationDesc;
+    private final String downstreamUrbanDesc;
+    private final String bankFullCapacityDesc;
+    private final String pumpBackLocationId;
+    private final String pumpBackOfficeId;
```

Comment:
Curious what you think about this being just these two fields.  Or should it be a Location object?

Reply:
hmm, I can see that going either way. But the full Location could be really convenient. Assuming there isn't a bad performance hit, but I wouldn't expect so since the database contains office,location and that query is usually rather quick.

Reply:
I think location ref is a good idea if you don't want to go full Location, for the Embankments that I'm working on, I have two fields projectId and projectOfficeId that could be modified to projection-location-ref.

base-location-id
sub-location-id
location-id(?)
office-id

Reply:
I'm going to see how far I can get with Location.  I may still switch to some sort of LocationRef.  Surprised we haven't needed a LocationRef until now

Reply:
Sounds reasonable, if we go to the point of "LocationRef" we'll want to revisit talks about single unique identifier and how that gets looked up. e.g. `/locations/<LocationRef>`  where location ref is something that can be in the path correctly.